lib/datadog/statsd/serialization/stat_serializer.rb

```diff
@@ -37,9 +37,18 @@ def global_tags
         attr_reader :prefix
         attr_reader :tag_serializer
 
-        def formatted_metric_name(metric_name)
-          formatted = Symbol === metric_name ? metric_name.name : metric_name.to_s
+        if RUBY_VERSION < '3'
+          def metric_name_to_string(metric_name)
+            metric_name.to_s
+          end
+        else
+          def metric_name_to_string(metric_name)
+            metric_name.is_a?(Symbol) ? metric_name.name : metric_name.to_s
```

Comment:
Minor: `Symbol === metric_name` as it was before may be slightly more efficient :thinking: